iyzico-payment-module/class-wc-gateway-iyzico-checkout-form.php

```diff
@@ -390,21 +395,25 @@ function check_iyzicocheckoutform_response() {
 
                         $installment = $response->getInstallment();
                         if (!empty($installment) && $installment > 1) {
-                            $installment_fee = $response->getPaidPrice() - $response->getPrice();
-                            $order_fee = new stdClass();
-                            $order_fee->id = 'Installment Fee';
-                            $order_fee->name = __('Installment Fee', 'iyzico-woocommerce-checkout-form');
-                            $order_fee->amount = floatval($installment_fee);
+
+                            $installment_fee    = $response->getPaidPrice() - $response->getPrice();
+                            $installment_fee    = esc_sql($installment_fee);
```

Comment:
sonucu neden escape ettik? Ayrıca değerin float olması gerektiği için escape etmeye gerek yok. floatval() fonksiyonu zaten string'i float a çevirir.